src/configs/shared.ts

```diff
@@ -20,3 +20,8 @@ export const DEFAULT_BOT_DELAY = 100; // 100ms
  */
 export const DEFAULT_FOLLOWUP_TIME = "4 days"; // 4 days
 export const DEFAULT_DISQUALIFY_TIME = "7 days"; // 7 days
+
+export const DEFAULT_CHAIN_ID = 1;
+export const DEFAULT_RPC_ENDPOINT = "https://cloudflare-eth.com";
```

Comment:
```suggestion
export const DEFAULT_RPC_ENDPOINT = "https://rpc-pay.ubq.fi/v1/mainnet";
```

I want to make dedicated RPC endpoints for all of our applications because its simple, free, and we can isolate logging metrics in case of abuse. I made this one the other day: 

https://github.dev/ubiquity/pay.ubq.fi/blob/773947bad8fd069adf99fb223373fcbdfab4600d/static/scripts/get-contract.js#L3

Reply:
I haven't settled on a scalable naming formula but ideally a standardized way to name all these dedicated RPC endpoints.